kuma/wiki/tests/test_views_code.py

```diff
@@ -87,18 +87,16 @@ def test_code_sample_host_allowed(code_sample_doc, settings, client):
 
 # The pytest-django urls marker also resets urlconf caches after the test
 @pytest.mark.urls(settings.ROOT_URLCONF)
-def test_code_sample_host_restricted_host(code_sample_doc, settings, client):
+def test_code_sample_host_restricted_host(code_sample_doc, constance_config,
+                                          settings, client):
     """Users are allowed to view samples on the attachment domain."""
-    # Render the document, which will be a sync operation, before applying
-    # host restrictions.
-    code_sample_doc.get_rendered()
-
     url = reverse('wiki.code_sample',
                   args=[code_sample_doc.slug, 'sample1'])
     host = 'sampleserver'
     settings.ALLOWED_HOSTS.append(host)
     settings.ATTACHMENT_HOST = host
     settings.ENABLE_RESTRICTIONS_BY_HOST = True
+    constance_config.KUMASCRIPT_TIMEOUT = 1
```

Comment:
Why this change? Is that to deliberately cause a KS timeout?

Reply:
Without this change, kumascript rendering is skipped (see https://github.com/mozilla/kuma/blob/aaabab98c62ebde4b4088055967568b4d9971b86/kuma/wiki/models.py#L572), so the call to `kuma.wiki.kumascript.get`, and eventually `document.get_absolute_url()`, is never made, and the problem is never exposed.

Reply:
It's so odd looking so could you please explain what you just explained to me in a code comment? Or do you think that's dumb because it's something everyone knows?

Reply:
I agree, and I've already added a comment, as you suggested, thanks!